pages/common/vim.md

```diff
@@ -1,31 +1,32 @@
 # vim
 
-> Vi IMproved, a programmer's text editor.
+> Vi IMproved, a programmer's text editor, that follows the modal paradigm.
```

Comment:
Heheh, the blessing and the curse of vim. Well, I tried to give a glimpse at this in the second line of the description, but yeah, this could be worded in a simpler way. Let me give it a try.